perun/templates/postprocess.__init__.jinja2

```diff
@@ -4,7 +4,3 @@ TODO: Write long docstring of the new postprocessor
 """
 
 SUPPORTED_PROFILES = ["{{ supported_types|join('\", \"') }}"]
-
-{% if author is defined %}
```

Comment:
This is actually bound to a certain parameter in CLI in (I believe) utils_group? Can you remove the parameter as well in this PR? Maybe you did, but there is so much changes, that I just skimmed it.

Reply:
Oh, good catch. I think I found it.

Reply:
The new commit should contain a fix.

Reply:
Good, feel free to merge.